src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: $main-font;
```

Comment:
Ensure that the variable `$main-font` is defined somewhere in your SCSS files. If it's not defined, it will cause a compilation error.